components/server/src/api/workspace-service-api.ts

```diff
@@ -63,4 +67,20 @@ export class WorkspaceServiceAPI implements ServiceImpl<typeof WorkspaceServiceI
             yield response;
         }
     }
+
+    async listWorkspaces(req: ListWorkspacesRequest, context: HandlerContext): Promise<ListWorkspacesResponse> {
+        const { limit } = parsePagination(req.pagination, 50);
```

Comment:
(nit) do we need some checks here? is organizationId required? if so please use uuidValidate like in https://github.com/gitpod-io/gitpod/pull/19038/files/6af59241d060511346114005a62ae2f4ff1a5548#diff-0aca56a8d97089c4a315ca83b1192be9aa1a08a46bc3f2f67a9b115ce7efde56R65

but please check I'm no sure

Reply:
Added, but not sure about the error message. i.e. In js, the field is defined as `organizationId` but in golang it's named `OrganizationId` and there may be more language's SDKs.

Reply:
let's use organizationId